modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerSelfTest.java

```diff
@@ -2067,6 +2070,71 @@ void testGetIndexesForTableInSortedOrderById() {
         assertThat(tableIndexIds(catalogVersion, tableId(TABLE_NAME)), equalTo(List.of(indexId0, indexId1, indexId2)));
     }
 
+    @Test
+    void testTableRename() {
+        createSomeTable(TABLE_NAME);
+
+        int prevVersion = manager.latestCatalogVersion();
+
+        int tableId = manager.tables(prevVersion).iterator().next().id();
+
+        CatalogCommand command = RenameTableCommand.builder()
+                .schemaName(SCHEMA_NAME)
+                .tableName(TABLE_NAME)
+                .newTableName(TABLE_NAME_2)
+                .build();
+
+        CompletableFuture<Void> renameFuture = manager.execute(command);
```

Comment:
Do we need this variable? Could it be just inlined?

Reply:
fixed